core/src/main/java/org/elasticsearch/action/bulk/BulkProcessor.java

```diff
@@ -79,6 +78,7 @@
         private int bulkActions = 1000;
         private ByteSizeValue bulkSize = new ByteSizeValue(5, ByteSizeUnit.MB);
         private TimeValue flushInterval = null;
+        private BackoffPolicy backoffPolicy = BackoffPolicy.noBackoff();
```

Comment:
yeah, I meant that we do the right thing and back off by default. Can you please change it? sorry for being late..